packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/modelparts/ModelMetadata.java

```diff
@@ -76,6 +79,11 @@ public class ModelMetadata extends SupportAdditionalProperties implements Serial
 	@TSOptional
 	JsonNode source;
 
+	@TSOptional
+	@Lob
+	@JdbcTypeCode(Types.BINARY)
+	private byte[] description;
+
```

Comment:
Probably related to this? I uploaded a model json I had locally and saw this error once I got into its model page

```
2024-09-25 09:44:43 [INFO] REQUEST STARTED | MiraController.convertAMRtoMMT | method=POST; uri=/mira/amr-to-mmt?project-id=3d7b0bff-5c55-48b4-bc4c-0dd7aa876967;  [software.uncharted.terarium.hmiserver.interceptors.RequestLoggingInterceptor:40]
2024-09-25 09:44:43 [ERROR] Unable to serialize input [software.uncharted.terarium.hmiserver.controller.mira.MiraController:135]
com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot access contents of TextNode as binary due to broken Base64 encoding: Illegal character '<' (code 0x3c) in base64 content (through reference chain: software.uncharted.terarium.hmiserver.models.dataservice.model.Model["metadata"]->software.uncharted.terarium.hmiserver.models.dataservice.modelparts.ModelMetadata["description"])
	at com.fasterxml.jackson.databind.exc.InvalidFormatException.from(InvalidFormatException.java:67)
	at com.fasterxml.jackson.databind.node.TextNode.getBinaryValue(TextNode.java:78)
	at com.fasterxml.jackson.databind.node.TreeTraversingParser.getBinaryValue(TreeTraversingParser.java:343)
	at com.fasterxml.jackson.databind.deser.std.PrimitiveArrayDeserializers$ByteDeser.deserialize(PrimitiveArrayDeserializers.java:469)
```

[aaaaa.json](https://github.com/user-attachments/files/17131888/aaaaa.json)

Reply:
I'm unable to replicate the issue. And that with a brand new install of the application.

Reply:
This error happens when I was typing into the description's text editor

Reply:
I see the error is in the MIRA controller.

Reply:
Removed model description from the serializing